### PR TITLE
fix CI build

### PR DESCRIPTION
**Pull request by a-tarasyuk**

### CODE REVIEW

**Comment by jakebailey** [comment]
Fwiw I think we do normalize newlines sometimes in fourslash, if that's part of the issue.

**Comment by a-tarasyuk** [comment]
@jakebailey The issue is to preserve the new line, from the target file. This PR https://github.com/microsoft/TypeScript/pull/54164 conflicts with https://github.com/microsoft/TypeScript/pull/54358 which was merged before.

File: tests/cases/fourslash/moveToFile_overloads1.ts [hunk 1]
```diff
@@ -14,7 +15,8 @@ verify.moveToFile({
     newFileContents: {
         "/a.ts": "",
         "/add.ts":
-`function add(x: number, y: number): number;
+`
```
Comment (a-tarasyuk) [inline tests/cases/fourslash/moveToFile_overloads1.ts:18]: I mean this new line that exists in the target file. I added the `@newline` option to be sure that files use the same format. Does that make sense?

Comment (jakebailey) [reply tests/cases/fourslash/moveToFile_overloads1.ts:18]: The thing I was confused about is whether or not this relies on the new lines in the git checkout. Typically we use CRLF so I just didn't want things to fail in different ways. That's what I meant when I was referring to normalization; there are some fourslash calls which normalize things for this reason.

(But, I'm not quite at my desk so will give this a proper look in a bit)

**Review by sandersn — APPROVED** [review]

**Comment by sandersn** [comment]
Passes on my machine; I'll merge this when CI finishes.

**Review by jakebailey — APPROVED** [review]

**Pull request merged**